yamale/validators/tests/test_validate.py

```diff
@@ -142,3 +142,23 @@ def test_mac():
     assert not v.is_valid('ab:cd:ef:12:34:56:78')
```

Comment:
It would also be good to add some YAML tests with actual examples of usage in it.